sympy/functions/elementary/integers.py

```diff
@@ -29,9 +32,7 @@ def eval(cls, arg):
                 return cls(i)*S.ImaginaryUnit
             return cls(arg, evaluate=False)
 
-        v = cls._eval_number(arg)
-        if v is not None:
-            return v
+
```

Comment:
I think there should only be one blank line here

Reply:
Removed it.